src/main/java/com/leverx/springapp/demo/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.leverx.springapp.demo.controller;
+
+import com.leverx.springapp.demo.model.User;
+import com.leverx.springapp.demo.repository.UserRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private UserRepository userRepository;
+
+    public UserController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @PostMapping
+    public ResponseEntity createUser(@Valid @RequestBody User user) {
```

Comment:
@blackavgn i'm totally fine with 'ResponseEntity'  however:
1. ResponseEntity is a container. Please clarify return value in <>
2. We don't need such flexibility for all endpoints. Let's discuss it